SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md

```diff
@@ -2,15 +2,15 @@
 
 # Tutorials and guides
 
-## Offloading example project
+## Offloading Example Project
+> **This tutorial uses the Example Project from the GDK's [setup guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro).**</br>
 
-The UnrealGDK Example project contains a map called `CrashBot_Gym` that demonstrates how to set up a gameplay feature using offload to separate workers.
+The UnrealGDK Example project contains a map called `CrashBot_Gym` that demonstrates how to offload gameplay features to separate workers.
```

Comment:
Hey, I used **set up a feature using offloading** to be consistent with one of our topic titles.